tests/vmi_configuration_test.go

```diff
@@ -1877,9 +1877,16 @@ var _ = Describe("[sig-compute]Configurations", func() {
 	})
 
 	Context("with a custom scheduler", func() {
+		withScheduler := func(schedulerName string) libvmi.Option {
```

Comment:
Defining a function as a Lambda suggest to the reader that it changes during the program execution, or that it depend on a variable in their scope of definition. Neither is true in this case, so please use the normal `func withSchedulerName` syntax. Also notice the suggested name which is similar to a k8s API function.

Reply:
Done